src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -347,42 +322,79 @@ class NotebookEditor extends Component {
         }, [icon('times-circle', { size: 25 })]),
         saving && spinnerOverlay
       ])
-    }
-
-    const isCreating = clusterStatus === 'Creating'
-    const currentStep = clusterStatus !== 'Running' ? 1 : 2
-
-    const step = (index, text) => div({ style: styles.step.container }, [
-      div({ style: styles.step.col1 }, [
-        index < currentStep && icon('check', { size: 24, style: { color: colors.green[0] } }),
-        index === currentStep && (failed ? icon('times', { size: 24, style: { color: colors.red[0] } }) : spinner())
-      ]),
-      div({ style: styles.step.col2 }, [text])
-    ])
+    } else {
+      const isCreating = clusterStatus === 'Creating'
+      const isRunning = clusterStatus === 'Running'
+      const currentStep = isRunning ? 2 : 1
+
+      const makeStep = (index, shortText, fullText) => {
```

Comment:
@katevoss and I were chatting about ideas to maybe make it clearer that these are steps. I think it could be useful to add 2 dots in between each step. It adds clarity that these are steps that happen one after the other, and it also mimics the "Script/Inputs/Outputs/Run Analysis" flow in the workflow view. 

thoughts?

Reply:
Since there are only two steps, and the second is typically pretty fast, I wonder how much value users actually get out of seeing them laid out. Perhaps we could just show a single message with A) what's currently happening, and B) how long we estimate until the notebook is ready.